app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -24,7 +24,11 @@ class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer
 
   def provider_object(connection = nil)
     connection ||= ext_management_system.connect
-    connection.bucket(ems_ref)
+    region = connection.client.get_bucket_location(:bucket => ems_ref).location_constraint
+    region = "us-east-1" if region.empty? # SDK returns empty string for default region
```

Comment:
Alternatively (perhaps even more clean to me), you could have a `def connect` in the `cloud_object_store_container` that would do 

```
    region = connection.client.get_bucket_location(:bucket => ems_ref).location_constraint
    region = "us-east-1" if region.empty? # SDK returns empty string for default region
    ext_management_system.connect(:region => region)
```